social_website/views.py

```diff
@@ -522,3 +525,15 @@ def signup_view(request, template_name='social_website/signup.html',
 
     return TemplateResponse(request, template_name, context,
                             current_app=current_app)
+
+def save_email_and_get_data(request):
+    email = request.GET.get("email", "")
+    filename = 'connector_emails.txt'
+    if os.path.exists(filename):
+        append_write = 'a'  # append if already exists
+    else:
+        append_write = 'w'  # make a new file if not:
+    file_object = open(filename, append_write)
+    file_object.write("Email: " + email + '\n')
+    file_object.close()
+    return JsonResponse({"data": "https://github.com/digitalgreenorg/farmstack-open"})
```

Comment:
Change the url to https://docs.farmstack.co/deploying-farmstack/setting-up-video-library-connector